chefignore

```diff
@@ -92,3 +92,10 @@ Vagrantfile
 # Travis #
 ##########
 .travis.yml
+
+# chef-provisioner
+.chef/*.pem
+.chef/local-mode-cache/
+/clients/
+/nodes/
+/vms/
```

Comment:
We may also want to ignore `controller_compute.rb`, `vagrant_linux.rb`, and the other chef provisioning files.

Which brings up the question, do we want to start putting all of the chef provisioning files in one dir?